doc/mgr/nfs.rst

```diff
@@ -98,7 +58,7 @@ cluster)::
 
     "2 host1,host2"
 
-To deploy NFS with an HA front-end (virtual IP and load balancer), add the
+To deploy NFS with a high-availability front-end (virtual IP and load balancer), add the
 ``--ingress`` flag and specify a virtual IP address. This will deploy a combination
 of keepalived and haproxy to provide an high-availability NFS frontend for the NFS
 service.
```

Comment:
Add link to https://docs.ceph.com/en/latest/cephadm/nfs/#high-availability-nfs ?